From 3df1f1d6dc16456a7a2f9c8b5b22708e239ce3ef Mon Sep 17 00:00:00 2001
From: "Liu,Dalei" <daleix.liu@intel.com>
Date: Wed, 8 Jul 2015 22:58:45 +0800
Subject: [PATCH 445/462] Revert "dw9718: Add power down sequence"

This reverts commit a57e6cc1e1a59d1e30781c12e9d1af05833d2da6.

Change-Id: If9c6ceb6a246eab2cf2c3f834e5d27bfe6bc34dc
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-40956
Signed-off-by: Liu,Dalei <daleix.liu@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/64097
Approver: Yunliang Ding <yunliang.ding@intel.com>
Reviewed-by: Yunliang Ding <yunliang.ding@intel.com>
Reviewed-by: Soren Friis <soren.friis@intel.com>
Validation-Android: Wei Tao <wei.tao@intel.com>
Integrator: WeiX Meng <weix.meng@intel.com>
Build: WeiX Meng <weix.meng@intel.com>
Reviewed-by: WeiX Meng <weix.meng@intel.com>
Maintainer: WeiX Meng <weix.meng@intel.com>
---
 drivers/external_drivers/camera/drivers/media/i2c/imx/dw9718.c | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/imx/dw9718.c b/drivers/external_drivers/camera/drivers/media/i2c/imx/dw9718.c
index 1cf2cb3..ec34e6e 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/imx/dw9718.c
+++ b/drivers/external_drivers/camera/drivers/media/i2c/imx/dw9718.c
@@ -113,11 +113,6 @@ int dw9718_vcm_power_up(struct v4l2_subdev *sd)
 		dev_err(&client->dev, "DW9718_PD power_ctrl failed %d\n", ret);
 		return ret;
 	}
-	ret = dw9718_i2c_wr8(client, DW9718_PD, 0);
-	if (ret < 0) {
-		dev_err(&client->dev, "write DW9718_PD to 0 failed %d\n", ret);
-		goto fail_powerdown;
-	}
 	/* Wait for VBAT to stabilize */
 	udelay(100);
 
@@ -178,11 +173,6 @@ int dw9718_vcm_power_down(struct v4l2_subdev *sd)
 	if (!dw9718_dev.power_on)
 		return 0;
 
-	ret = dw9718_i2c_wr8(client, DW9718_PD, 1);
-	if (ret)
-		dev_err(&client->dev, "%s: write DW9718_PD to 1 failed\n",
-				__func__);
-
 	ret =  dw9718_dev.platform_data->power_ctrl(sd, 0);
 	if (ret) {
 		dev_err(&client->dev, "%s power_ctrl failed\n",
-- 
1.9.1

